src/routes/Projects/components/Projects.tsx

```diff
@@ -27,6 +28,7 @@ export default class Projects extends React.PureComponent<IProjectProps, {}> {
     const { projects } = this.props;
     return (
       <div className="projects-container">
+        <FixMeMetas title="Fixme | Projects" description="projects page meta description"/>
```

Comment:
The description isn't really descriptive, maybe something like "Projects included in the FixMe issue listing" (or something similar) would be clearer